### DISCUSSION TITLE
AI 打入 IC 設計!

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#816] [General]
目前我們對於AI能夠在業界的應用大多數都是可以取代基礎勞力/重複性高/不需高階知識的工作，但聯發科利用AI開發了能夠設計晶片的機器，讓機器也能夠做工程師的工作，若未來被廣泛應用，應該會對業界產生許多正負面的影響，也讓AI的發展跨進了一大步
https://buzzorange.com/techorange/2022/10/25/mediatek-ic-chip-upgrade/

**Comment by a45320614** [comment]
這項技術如果能實際應用對於台灣IC設計產業會帶來相當大的衝擊吧，IC設計可以說是許多工程師的夢想工作，雖然這項技術為台灣帶來了技術上的進步，不過也會造成許多工程師的失業問題

**Comment by Jasper525** [comment]
現在台灣的各大IC設計廠也有在招AI工程師了，可見人工智慧對科技發展的助益。

**Comment by ching020721** [comment]
AI應用在IC Design，結合EDA跨域合作可以在短時間產生最佳化電路，不僅提高技術還能使研發人力去做更複雜的系統，但可能還要考量成本上的問題，還有需要精進的地方肯定也很多，若能慢慢成長絕對會是IC設計產業的一大助益。

**Comment by Pietra1226** [comment]
國外有AI做EDA，設計過程中也一直不斷training，幾乎可以24小時持續運作，這樣遲早超過人工設計，看來這對IC業有很大衝擊。

**Comment by WEI-CHEN-CHUN** [comment]
未來工程師主要的方向會變成寫AI來設計IC?

**Comment by Chiway1229** [comment]
台灣是個以IC設計產業為主體的國家，如果真的有這項技術，應該對經濟是蠻大的打擊

**Comment by StevenWanglolz** [comment]
隨著AI不斷地在學習，未來真的會有越來越多職業會被取代呢~
